tests/testAstrometrySourceSelector.py

```diff
@@ -17,17 +18,17 @@
 #
 # You should have received a copy of the LSST License Statement and
 # the GNU General Public License along with this program.  If not,
-# see <https://www.lsstcorp.org/LegalNotices/>.
+# see <http://www.lsstcorp.org/LegalNotices/>.
 #
```

Comment:
Looks like you are not using he current copyright template: https://github.com/lsst/templates/blob/master/CopyrightHeader.py